test/models/bot/alegre_4_test.rb

```diff
@@ -0,0 +1,65 @@
+require_relative '../../test_helper'
+
+class Bot::Alegre4Test < ActiveSupport::TestCase
+  def setup
+    super
+    ft = DynamicAnnotation::FieldType.where(field_type: 'language').last || create_field_type(field_type: 'language', label: 'Language')
+    at = create_annotation_type annotation_type: 'language', label: 'Language'
+    @field = create_field_instance annotation_type_object: at, name: 'language', label: 'Language', field_type_object: ft, optional: false
+    @bot = create_alegre_bot(name: "alegre", login: "alegre")
+    @bot.approve!
+    p = create_project
+    p.team.set_languages = ['en','pt','es']
+    p.team.save!
+    @bot.install_to!(p.team)
+    @team = p.team
+  end
+
+  def teardown
+    super
+  end
+
+  test "should not send bad text for langid" do
+    text = "instagram-cekfakta-2023-25562004"
+    pm1 = create_project_media team: @team, quote: text
+    Bot::Alegre.stubs(:get_language_from_alegre).returns("en")
+    assert_equal Bot::Alegre.get_language_from_text(pm1, text), "und"
+    Bot::Alegre.unstub(:get_language_from_alegre)
+  end
+
+  test "should not send blank text for langid" do
+    text = ""
+    pm1 = create_project_media team: @team, quote: text
+    Bot::Alegre.stubs(:get_language_from_alegre).returns("en")
+    assert_equal Bot::Alegre.get_language_from_text(pm1, text), "und"
+    Bot::Alegre.unstub(:get_language_from_alegre)
+  end
+
+  test "should send text for langid" do
+    text = "anything else"
```

Comment:
Let's change this to `text = "instagram-cekfakta-2023-25562004 with more should pass"

Reply:
changed the test to "instagram-cekfakta-2023-25562004 with more should pass" in the other good title tests too.